### DISCUSSION TITLE
Doris升级到1.2后，group_concat(xx order by xx) 报 “ORDER BY for arguments only support in vec exec engine” 如何解决？

### DISCUSSION THREAD

**Discussion by JiaJunWu2022** [apache/doris#15478] [A - General / Q&A] [answered]
表结构如下：
`CREATE TABLE `TEST1` (
  `no` varchar(50) NOT NULL COMMENT '编号',
  `f_name` varchar(50) NOT NULL COMMENT '姓',
  `l_name` varchar(2000) NOT NULL COMMENT '名'
) ENGINE=OLAP
DUPLICATE KEY(`no`)
COMMENT '业务数据'
DISTRIBUTED BY HASH(`no`) BUCKETS 8
PROPERTIES (
"replication_allocation" = "tag.location.default: 1",
"in_memory" = "false",
"storage_format" = "V2",
"disable_auto_compaction" = "false"
);`

执行语句如下：
`select no,f_name,group_concat(distinct l_name order by l_name) gc_l_name from TEST1 group by no,f_name`

执行结果如下：
`[Err] 1105 - errCode = 2, detailMessage = ORDER BY for arguments only support in vec exec engine`

**Comment by LemonLiTree** [comment, marked as answer]
show variables like '%vec%'; 
看下向量化开了没有，没有的话开一下试试。

**Reply by JiaJunWu2022** [reply]
大神威武，就是这个问题，已经解决了，谢谢！！！
